src/subscription/mapAsyncIterator.js

```diff
@@ -39,8 +39,9 @@ export default function mapAsyncIterator<T, U>(
 
   let mapReject;
   if (rejectCallback) {
-    mapReject = error =>
-      asyncMapValue(error, rejectCallback).then(iteratorResult, abruptClose);
+    mapReject = error => rejectCallback ?
```

Comment:
A comment here would be helpful. Without `git blame`-ing this to see why the check is there, it's not obvious that it's for Flow.